src/dynamic_boundary_conditions/rainfall_data_from_hirds.py

```diff
@@ -111,14 +112,22 @@ def get_layout_structure_of_data(site_data: str) -> List[Tuple[int, float, str]]
                 rcp.append(float(rcp_result[0]))
                 time_period.append(period_result[0])
             else:
-                rcp.append(float('nan'))
+                rcp.append(float("nan"))
                 time_period.append(None)
-    # Merge the three different lists into one list of tuples
-    layout_structure = list(zip(skip_rows, rcp, time_period))
+            # Assign category to list
+            if re.search(r"standard error", line):
+                category.append("hist_stderr")
+            elif re.search(r"Historical Data\s$", line):
+                category.append("hist")
+            else:
+                category.append("proj")
```

Comment:
Can we remove these regular expressions for simpler form. Only if you agree and if it is correct.
```suggestion
            if "standard error" in line:
                category.append("hist_stderr")
            elif "Historical Data" in line:
                category.append("hist")
            else:
                category.append("proj")
```

Reply:
changed.